addons/sale_timesheet/controllers/portal.py

```diff
@@ -1,19 +1,23 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
+from werkzeug.exceptions import NotFound
+
 from odoo import http, _
 from odoo.exceptions import AccessError, MissingError
 from odoo.http import request
 from odoo.osv import expression
 
-from odoo.addons.account.controllers import portal
+from odoo.addons.account.controllers.portal import PortalAccount
 from odoo.addons.hr_timesheet.controllers.portal import TimesheetCustomerPortal
+from odoo.addons.portal.controllers.portal import pager as portal_pager
+from odoo.addons.project.controllers.portal import ProjectCustomerPortal
 
 
-class PortalAccount(portal.PortalAccount):
+class PortalProjectAccount(PortalAccount, ProjectCustomerPortal):
```

Comment:
Why not directly inheriting to only `ProjectCustomerPortal`? 🤔

Reply:
As this controller is to get the invoices, we need to use _prepare_my_invoices_values method which is in PortalAccount.

Reply:
Oh indeed, sorry, thanks